src/main.cpp

```diff
@@ -39,13 +39,20 @@
 #endif
 #if defined( Q_OS_MACX )
 #    include "mac/activity.h"
+extern void qt_set_sequence_auto_mnemonic ( bool bEnable );
 #endif
 
 // Implementation **************************************************************
 
 int main ( int argc, char** argv )
 {
 
+    // mnemonic keys are disabled by default on MacOS the following function enables them
+    // by "mac" design they will not be shown with underline characters in the GUI
+#if defined( Q_OS_MACX )
```

Comment:
Could you move the comment inside the `#if`.

Could you also start new sentences in appropriate places to make the comment easier to read.

Reply:
Thanks for the comments, I have made the updates. Hope the text reads better now.

Reply:
You've still not split it into sentences.  There are three separate ones but no full stops.

Reply:
For me there were just two sentences. I have updated the text.

Reply:
> mnemonic keys are disabled by default on MacOS the following function enables them

This reads in multiple ways and hence needs punctuation

> mnemonic keys are disabled by default
> on MacOS the following function enables them

> mnemonic keys are disabled by default on MacOS
> the following function enables them

Also, as the line is fairly long, it's not clear whether the second line is part of the same sentance as the first due to the lack of punctuation.  Overall, it hampers reading and understanding.

Reply:
@pljones I have revised the text now. Are you checking the latest?

Reply:
Apparently not...  Sorry!